pkg/service/service.go

```diff
@@ -12,6 +13,9 @@ import (
 	"github.com/redhat-developer/odo/pkg/util"
 )
 
+const provisionedAndBoundStatus = "ProvisionedAndBound"
```

Comment:
I didn't understand this status, since service is just `provisioned` not bound to anything,

```
$ odo service list
NAME                         TYPE                         STATUS
cakephp-mysql-persistent     cakephp-mysql-persistent     ProvisionedAndBound
```

Reply:
This is part of the discussion we had with @jorgemoralespou .

The idea is provide some extra statuses, so the user can more information about the service provisioning.
For example, the user should no if the ServiceBinding has been completed, hence the `ProvisionedAndBound` status.
Also the `ProvisionedAndLinked` status means that the service has been linked into a component